### DISCUSSION TITLE
save and continue button in frontend

### DISCUSSION THREAD

**Discussion by pablogadino** [joomla/joomla-cms#27698] [2. Cold Features]
### Steps to reproduce the issue

open an article for edition in frontend

### Expected result

presence of a "save and continue" button, as in backend

### Actual result

only "save" button, and this close the editor form.

### System information (as much as possible)

joomlacontenteditor or tinymce editor, both same UI.

### Additional comments

This is bad in a big site when many authors and long articles.
In Joomla 1.0 there was those buttons

**Comment by ghost** [comment]
How was it done in Joomla 1.0 ?

Expected result:
do you want this #25757  ?

**Comment by pablogadino** [comment]
I expect Save Changes and stay in editor. As in dektop editor, as in backend in actual Joomla, or as in 1.0 Joomla frontend. 
Actually save changes, close editor and shows articles list.
Fo continue editing is necessary find the rticle, reopen and find where we are before.
This is frustrating for my authors when are writing long content and want periodically save to avoid losses. 
In backend this is a button for this, in frontend is not.

**Comment by ghost** [comment]
is it the same ? #22232

**Comment by pablogadino** [comment]
Not same 22232, but in that one an user says:
"In fact I have always wished we had 3 buttons in frontend: Save, Save & Close and a Preview button ".
Save button for save changes whithout close the edit form, this is my need.

**Comment by ghost** [comment]
nice feature to have

**Comment by oorzaak** [comment]
I agree that this would be very helpful; both the "Save and continue" button and the preview.

**Reply by alexandreelise** [reply]
AltiDigi might have a solution for everyone wanting Save, Save & Close,  Save & New, Save & Copy on frontend for Joomla 3 and Joomla 4  [AD - Frontend Edit Buttons](https://altidigi.com/store/3-ad-frontend-edit-buttons). Share your thoughts and comments on this. Enjoy! @pablogadino
